test/extended/tests-extension/Makefile

```
@@ -0,0 +1,75 @@
+.PHONY: list-test-names
+list-test-names: build #HELP Show current full test names
+	@$(TOOLS_BIN_DIR)/$(BINARY_NAME) list -o names
+
+.PHONY: clean-metadata
+clean-metadata: #HELP Remove 'codeLocations' from metadata JSON
+	@echo "Cleaning metadata (removing codeLocations)..."
+	@for f in $(METADATA)/*.json; do \
+		jq 'map(del(.codeLocations))' "$$f" > "$$f.tmp" && mv "$$f.tmp" "$$f"; \
+	done
+
+.PHONY: verify-metadata #HELP To verify that the metadata was properly updated
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)